instructors/instructor-notes.md

```diff
@@ -286,9 +286,9 @@ As written, this can run long.  Suggestions:
 
 ## VI. Teaching Demonstration Tips
 
-This section contains tips and tricks on demo hosting that may be contributed by any Trainer describing things that work for them. For official/technical guidelines on how to run a demo, see the [Handbook][demos].
+Here are suggestions from the Trainer Community for the smooth running of your teaching demo. For official/technical guidelines on how to run a demo, see the [Handbook][demos].
 
-Suggested points to make during introductions:
+Points to make during introductions:
```

Comment:
I like this shorter, more direct phrasing, but I would like a second opinion because I am sometimes abrasively direct.  @karenword is it important that we have "suggested" here to keep it clear that trainers can customize or is this revision okay?